examples/simple-example.rs

```diff
@@ -275,7 +319,7 @@ struct MyCircuit<F: FieldExt> {
 
 impl<F: FieldExt> Circuit<F> for MyCircuit<F> {
     // Since we are using a single chip for everything, we can just reuse its config.
-    type Config = FieldConfig;
+    type Config = FieldConfigEnum;
```

Comment:
```suggestion
    type Config = FieldConfig;
```

I think this could also maybe instead be `type Chip = FieldChip`, since I _think_ now we'll always just construct the chip from its config immediately inside `Circuit::synthesize`. But that's a higher-level refactor (and would probably also require making the layouter choice an associated type), so let's leave that out of this PR.